mne/viz/epochs.py

```diff
@@ -536,3 +809,201 @@ def plot_epochs_psd(epochs, fmin=0, fmax=np.inf, proj=False, n_fft=256,
     if show:
         plt.show()
     return fig
+
+
+def _plot_traces(params):
+    """ Helper for plotting concatenated epochs """
+    ax = params['ax']
+    n_channels = params['n_channels']
+    lines = params['lines']
+    data = params['data'] * params['scale_factor']
+    offsets = params['offsets']
+
+    length = len(params['epochs'].times)
```

Comment:
Well, actually n_samples would probably be the most accurate. Not sure what n_times means...